src/ServiceControl.Transports.ASB/ASBForwardingTopologyTransportCustomization.cs

```diff
@@ -24,6 +25,12 @@ static void ConfigureTransport(TransportExtensions<AzureServiceBusTransport> tra
             transport.Transactions(TransportTransactionMode.SendsAtomicWithReceive);
             transport.ConnectionString(transportSettings.ConnectionString);
             transport.Sanitization().UseStrategy<ValidateAndHashIfNeeded>();
+
+            transport.MessageReceivers().PrefetchCount(0);
+            transport.Queues().LockDuration(TimeSpan.FromMinutes(5));
+            transport.Subscriptions().LockDuration(TimeSpan.FromMinutes(5));
+            transport.MessagingFactories().NumberOfMessagingFactoriesPerNamespace(2);
+            transport.NumberOfClientsPerEntity(Math.Min(Environment.ProcessorCount, transportSettings.MaxConcurrency));
```

Comment:
Should we move these into a base class or a shared location? Or is the fact that they are currently identical a coincidence?

Reply:
I guess these are not topology-dependent so 👍 for a base class